keras/layers/advanced_activations.py

```diff
@@ -129,11 +129,8 @@ def build(self, input_shape):
 
     def call(self, inputs, mask=None):
         pos = K.relu(inputs)
-        if K.backend() == 'theano':
-            neg = (K.pattern_broadcast(self.alpha, self.param_broadcast) *
-                   (inputs - K.abs(inputs)) * 0.5)
-        else:
-            neg = -self.alpha * K.relu(-inputs)
+        alpha_broadcast = K.pattern_broadcast(self.alpha, self.param_broadcast)
```

Comment:
This is only relevant for Theano and should not be in the public API. The conditional `if K.backend() == 'theano':` is acceptable in this case.